modules/account-lib/test/unit/coin/hbar/transactionBuilder/transferBuilder.ts

```diff
@@ -242,15 +242,15 @@ describe('HBAR Transfer Builder', () => {
       txBuilder.fee({ fee: '1000000000' });
```

Comment:
Could you update the builder instances in this file to use `CoinTransferBuilder`, i.e., replace all `getTransferBuilder()` with `getCoinTransferBuilder()`. So that the refactored builder can get tested directly. It might make more sense to name this test file `CoinTransferBuilder.ts` as well.

Reply:
ahh gotcha, didn't realize you kept the name the same in the BuilderFactory